flutter_install_type.go

```diff
@@ -0,0 +1,279 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/v2/command"
+)
+
+const (
+	FVMName              = "fvm"
+	ASDFName             = "asdf"
+	ManualName           = "manual"
+	FVMCacheVersionsPath = "/fvm/versions"
+	FVMCacheDefaultPath  = "/fvm/default/bin/flutter"
+	ASDFShimsPath        = "/.asdf/shims/flutter"
+)
+
+type FlutterInstallType struct {
+	Name                     string
+	IsAvailable              bool                                          // if the tool is available, this will be set to true later
```

Comment:
Please use standard Go function comments please 

```suggestion
    // IsAvailable will be set to true later if the tool is detected on the host
	IsAvailable              bool
```